packages/core-mobile/app/new/routes/(signedIn)/(modals)/accountSettings/connectedSites.tsx

```diff
@@ -132,21 +127,6 @@ const ConnectedSitesScreen = (): JSX.Element => {
 
   const renderSeparator = (): JSX.Element => <View sx={{ height: 12 }} />
 
-  // const renderFooter = useCallback(() => {
-  //   if (allApprovedDapps.length > 1) {
-  //     return (
-  //       <Button
-  //         type="primary"
-  //         size="large"
-  //         style={{ marginBottom: bottomInset + 16 }}
-  //         onPress={() => disconnectAllDapps()}>
-  //         Disconnect all
-  //       </Button>
-  //     )
-  //   }
-  //   return <></>
-  // }, [allApprovedDapps.length, bottomInset, disconnectAllDapps])
-
```

Comment:
we do need the ability to disconnect all dApps at once. does the "disconnect all" live somewhere else now?